_source/_posts/2020-10-09-okta-spring-security-feature-flags.md

```diff
@@ -30,17 +30,17 @@ OpenID Connect (OIDC) rides on top of OAuth 2.0 for a modern Single Sign-on, aut
 
 > If you're interested in learning more about OIDC and OAuth 2.0, [here](https://developer.okta.com/blog/2017/07/25/oidc-primer-part-1) and [here](https://oauth.com) are good places to start. Look for more links to posts on OIDC and OAuth 2.0 at the end of this post.
 
-### Register for an Okta Org
+### Register for an Okta Org and Create an OIDC App
 
-Head on over to: [https://developer.okta.com/signup](https://developer.okta.com/signup). Fill out the form and click **Get Started**.
-
-You'll get an email from Okta to confirm your address. Click on **ACTIVATE MY ACCOUNT**. You'll next see an onboarding page in your Okta Org. You can skip this for now.
-
-Let's add a few users and groups for use later in the Spring Boot app.
+{% include setup/cli.md type="web" framework="Okta Spring Boot Starter"
+  loginRedirectUri="http://localhost:8080/login/oauth2/code/okta"
+  logoutRedirectUri="http://localhost:8080" %}
```

Comment:
These redirects are the defaults for the Okta Spring Boot starter, so you can remove them.